drinfomon.lic

```diff
@@ -508,7 +508,7 @@ CharSettings['vault_info'] = {} unless CharSettings['vault_info'].class == Hash
 
 kronar_banks = ['Crossings', 'Dirge', 'Leth Deriel', 'Ilaya Taipa']
 lirum_banks = ["Aesry Surlaenis'a", "Hara'jaal", "Mer'Kresh", "Muspar'i", 'Ratha', 'Riverhaven', "Rossman's Landing", 'Therenborough', 'Throne City']
-dokora_banks = ['Ain Ghazal', 'Boar Clan', "Chyolvea Tayeu'a", 'Hibarnhvidar', 'Shard']
+dokora_banks = ['Ain Ghazal', 'Boar Clan', "Chyolvea Tayeu'a", 'Hibarnhvidar', 'Shard', "Raven's Point"]
```

Comment:
Does `Inner Hibarnhvidar` need to be added to this list?

Reply:
There isn't one for the 2 banks that share a balance on Ratha so I assumed not